.github/workflows/main.yml

```diff
@@ -18,8 +18,8 @@ jobs:
       - name: Installing Dependencies
         run: npm install
       - name: Building
-        run: npm run build
-      - name: upload
+        run: npm ci
+      - name: Deploying
```

Comment:
`npm ci` should replace `npm install` and not `npm run build`.

Reply:
I have corrected this error